sw/device/silicon_creator/lib/BUILD

```diff
@@ -362,13 +362,24 @@ cc_library(
     ],
 )
 
+filegroup(
+    name = "sigverify_src_for_keys_unittest",
```

Comment:
nit: This is probably not needed since you are adding the files directly to the `sigverify_unittest` and `sigverify` test rules. I guess it is useful to have if the `sigverify` target is composed of multiple source files.